src/docs/content/docs/install/building.md

```diff
@@ -0,0 +1,58 @@
++++
```

Comment:
I'd rather put building from source last because it's the least likely to be used.  Apt & Yum are the things most people will do, then probably docker, then source.

Reply:
Fair call.